app/src/androidTest/java/ch/epfl/sdp/drone3d/ui/mission/MapActivityTest.kt

```diff
@@ -46,11 +58,17 @@ class MapActivityTest {
         Assert.assertEquals("ch.epfl.sdp.drone3d", appContext.packageName)
     }
 
-    @BindValue val authService: AuthenticationService = mock(AuthenticationService::class.java)
+    @BindValue
+    val authService: AuthenticationService = mock(AuthenticationService::class.java)
 
     @Before
     fun setUp() {
         Intents.init()
+        `when`(context.checkSelfPermission(ACCESS_FINE_LOCATION))
```

Comment:
Is it necessary to have it here ? If you redefine it for every test ?

Reply:
No, I've just left it here when I was trying some stuff. I will delete it.